user_api/auth.py

```diff
@@ -2,13 +2,13 @@
 from rest_framework.authtoken.models import Token
 from rest_framework.response import Response
 
-class CustomAuthToken(ObtainAuthToken):
-
+class CustomAuthToken(ObtainAuthToken,):
```

Comment:
Why this class is being created ? This is a PR for creating the CRUD Api's why are you doing authentication in it. I think there is different PR for authentication